products/public-websites/post-mortems/off-schedule-deployment-11-04-20.md

```diff
@@ -5,7 +5,7 @@ The VSA team Public Websites was set to launch a new section of the website call
 ## Background
 On November 4th, 2020, the VSA team Public Websites was set to launch a new section of the website called Resources and Support, living at `www.va.gov/resources/`. This new section contained Drupal-hosted content, a front-end search application, and some front-end code to generate certain static pages. This required quite a bit of planning and coordination between engineering and content to ensure everything that needed to be launched _was_ launched.
 
-In his original [pull request for launching](https://github.com/department-of-veterans-affairs/vets-website/pull/14832) the product, Nick Sullivan of Public Websites made an oversight by forgetting to remove a feature flag for building the R&S (Resources and Support) search application. This led to a follow-up [pull request](https://github.com/department-of-veterans-affairs/vets-website/pull/14845) and a request for an off-schedule deployment around 4 PM. Soon after requesting and while waiting for approval, an unrelated pull request was merged into vets-website master, causing master to fail, preventing off-schedule deployments. Nick then opened another [pull request](https://github.com/department-of-veterans-affairs/vets-website/pull/14850) to revert the changes in that other. Meanwhile, a content deployment published the new R&S articles into production while the R&S search application was still not published. This would mean R&S did not have a functional search for a short timeframe.
+In his original [pull request for launching](https://github.com/department-of-veterans-affairs/vets-website/pull/14832) the product, Nick Sullivan of Public Websites made an oversight by forgetting to remove a feature flag for building the R&S (Resources and Support) search application. This led to a follow-up [pull request](https://github.com/department-of-veterans-affairs/vets-website/pull/14845) and a request for an off-schedule deployment around 4 PM. Soon after requesting and while waiting for approval, an unrelated pull request was merged into vets-website main, causing main to fail, preventing off-schedule deployments. Nick then opened another [pull request](https://github.com/department-of-veterans-affairs/vets-website/pull/14850) to revert the changes in that other. Meanwhile, a content deployment published the new R&S articles into production while the R&S search application was still not published. This would mean R&S did not have a functional search for a short timeframe.
```

Comment:
```suggestion
In his original [pull request for launching](https://github.com/department-of-veterans-affairs/vets-website/pull/14832) the product, Nick Sullivan of Public Websites made an oversight by forgetting to remove a feature flag for building the R&S (Resources and Support) search application. This led to a follow-up [pull request](https://github.com/department-of-veterans-affairs/vets-website/pull/14845) and a request for an off-schedule deployment around 4 PM. Soon after requesting and while waiting for approval, an unrelated pull request was merged into vets-website master, causing master to fail, preventing off-schedule deployments. Nick then opened another [pull request](https://github.com/department-of-veterans-affairs/vets-website/pull/14850) to revert the changes in that other. Meanwhile, a content deployment published the new R&S articles into production while the R&S search application was still not published. This would mean R&S did not have a functional search for a short timeframe.
```
Postmortem